backend/src/cepik/cepik.service.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
You shouldn't need to disable this rule.
Also for the future, please try to use `// eslint-disable-next-line` rule, to limit the amount of affected lines